src/test-utils/dom/breadcrumb-group/index.ts

```diff
@@ -15,7 +15,7 @@ export default class BreadcrumbGroupWrapper extends ComponentWrapper {
    * @see findBreadcrumbLink
    */
   findBreadcrumbLinks(): Array<ElementWrapper> {
```

Comment:
Not sure `findBreadcrumbLinks` should return an item that is not a link, but I assume this would be a breaking change we don't want to make?

Maybe it would be good to add a note to the component's [Testing page](https://cloudscape.design/components/breadcrumb-group/?tabId=testing) to clarify this (the fact that the function will return the last item even if it's not a link).

Reply:
Yeah, it would be a breaking change (and from a purely semantic point of view it didn't really return a link before, as the last item was an `a` tag with no `href` or click handlers). I wonder whether deprecating this, and replacing it with `findBreadcrumbItems` (with the same implementation) could be clearer, what do you think?

Reply:
Sounds good to me (but we should probably seek further team consensus about this since it's an API change)

Reply:
discussed further in team, preference is to keep the naming but make the documentation clearer